armoryengine/Transaction.py

```diff
@@ -678,7 +750,14 @@ def unserialize(self, toUnpack):
       startPos = txData.getPosition()
       self.inputs     = []
       self.outputs    = []
+      self.witnesses  = []
       self.version    = txData.get(UINT32)
+      marker = txData.get(UINT8)
+      flag = txData.get(UINT8)
+      if marker == MARKER and flag == FLAG:
```

Comment:
maybe rename these as WITNESS_MARKER and WITNESS_FLAG across the board, it will be confusing a year down the road when no one is thinking about segwit when reviewing code